src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class FigureSupplier extends ColorSupplier {
+    public static final int Max_Size = 10;
+    public Figure getRandomFigure() { 
+    ColorSupplier color = new ColorSupplier();
+    String randomColor = color.getRandomColor();
```

Comment:
The `getRandomColor` method returns a `String`, but it should return a `Color` enum constant. Update the `ColorSupplier` to return a `Color` instead of a `String`.